src/Core/Package.vala

```diff
@@ -626,13 +625,28 @@ public class AppCenterCore.Package : Object {
     public string? get_description () {
         if (description == null) {
             description = component.get_description ();
+
             if (description == null) {
                 if (backend_details == null) {
                     populate_backend_details_sync ();
                 }
 
                 description = backend_details.description;
             }
+
```

Comment:
Add a `description == null` check here and return `null` if so. It's possible that a package still won't have a description even after trying two different ways to get it.

Reply:
Worth noting that this change would resolve the segfault that @danrabbit was seeing when opening the "OS Updates" component while AppCenter was still loading.